packages/workbox-webpack-plugin/test/node/e2etest.js

```diff
@@ -140,6 +134,69 @@ describe(`Tests for webpack plugin`, function() {
   {
     "url": "styles/stylesheet-2.css",
     "revision": "\\w*"
+  },
+  {
+    "url": "webpackEntry.js",
+    "revision": "\\w*"
+  }
+\\]\\);`);
+      const result = regex.exec(swContents);
+      expect(result).to.exist;
+    });
+  });
+
+  it('should append publicPath to webpack generated files', () => {
+    fsExtra.copySync(
+      path.join(__dirname, '..', '..', '..', 'workbox-cli', 'test', 'static', 'example-project-1'),
+      tmpDirectory);
+
+    process.chdir(tmpDirectory);
+
+    const webpackConfig = {
+      entry: {
+        webpackEntry: path.join(tmpDirectory, 'webpackEntry.js'),
+      },
+      output: {
+        path: tmpDirectory,
+        filename: '[name].js',
```

Comment:
I tried addressing fixing all comments except this, I feel multi level can be achieved only by specifying configuration in user land. not sure about a good way of testing it and also about the specific thing to be tested.

We're mainly appending the public path which is already tested, any path beyond that would come from user land.